Templates (#173)
decidim/documentation: docs/en/modules/admin/pages/templates/proposal_answers.adoc

= Proposal answers templates

In this section, you can create, edit, and delete proposal answers templates, that you can then use when answering proposals. 

== Create a proposal answers template

NOTE: To be able to create a proposal answers template, you need to have created at least one 
xref:admin:components/proposals.adoc[proposals] component.

> To create a new block proposal answers template, you need to click the "New template" button. 

Reviewer: ```suggestion
To create a new proposal answers template, you need to click the "New template" button. 
```

image::templates/templates_proposal_answers_choice_component.png[Backend: Proposal component choice for template]

Choose the participatory space and component for which you want to create proposal answers to see the template answer form. 

image::templates/templates_proposal_answers_backend_form.png[Backend: new proposal answer template form]

.New proposal answer template form
|===
|Field |Type |Description

|Name
|Required

> |Title of this answer template. Choose a clear title, describing precisely what the answer is for, like "accepted proposal."

Reviewer: ```suggestion
|Title of this answer template. Choose a clear title, describing precisely what the answer is for, like "accepted proposal".
```

|Description
|Required
|Content of the answer provided to the proposals for which you choose this template. You can use variables in your answer. 

|Proposal state
|Required
|By default, you can choose Evaluating, Accepted, or Rejected. 

|===

== Manage

> All the templates appear in the questionnaire templates list:

Reviewer: ```suggestion
All the templates appear in the proposal answers templates list:
```

* *Template*: name of the proposal answer template.
* *Internal state*: state given to proposals when using this template. 
* *Add constraint*: proposal component where you can use this template. 
* *Created at*: date of creation.

> You can edit a template and its questions at any moment. 

Reviewer: ```suggestion
You can edit a template and its fields at any moment. 
```
